Add documentation link (#60)
Microsoft/MSBuildLocator: README.md

[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/Microsoft/MSBuildLocator/MSBuildLocator-ci)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=80)

# Documentation

> The documentation is located on the official Microsoft documentation site, in [this](https://docs.microsoft.com/en-us/visualstudio/msbuild/updating-an-existing-application#use-microsoftbuildlocator) page.

Reviewer: ```suggestion
Documentation is located on the official Microsoft documentation site: [Use Microsoft.Build.Locator](https://docs.microsoft.com/visualstudio/msbuild/updating-an-existing-application#use-microsoftbuildlocator).
```
